src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContextFactory.cs

```diff
@@ -188,8 +188,7 @@ public JobHostContextFactory(IStorageAccountProvider storageAccountProvider, ICo
                         queueConfiguration, exceptionHandler, trace, functions,
                         functionInstanceLogger, functionExecutor);
 
-                    Guid hostInstanceId = Guid.NewGuid();
```

Comment:
Yeah, I agree. I'm just pointing out that there is a non-zero chance that someone might get bit by this. But if they do, they're really doing an anti-pattern. Also for core SDK this is a new major release, so we can make a break here.